superset/assets/javascripts/components/FilterableTable/FilterableTable.jsx

```diff
@@ -0,0 +1,192 @@
+import { List } from 'immutable';
+import React, { PropTypes, PureComponent } from 'react';
+import {
+  Column,
+  Table,
+  SortDirection,
+  SortIndicator,
+} from 'react-virtualized';
+import { getTextWidth } from '../../modules/visUtils';
+
+require('./FilterableTableStyles.css');
```

Comment:
I think you should be able to `import` this? you might have to add a `.css` loader to weback but I thought that was set up in Superset.